### DISCUSSION TITLE
Can a Local IndexDB install be transferred to a Postgres Server Database version?

### DISCUSSION THREAD

**Discussion by SeanBannister** [lobehub/lobehub#4795] [Self Hosting] [answered]
I'm running a Local IndexDB version of LobeChat and was wonder if there was any way to convert my data over to the Server Database version of LobeChat?

**Comment by arvinxx** [comment, marked as answer]
Export you config and then import them to the server db version

**Reply by SeanBannister** [reply]
Awesome, thanks. For anyone else looking for this option, click on the icon in top left corner.

Out of interest, is it possible to go the other way, from a DB version to local? I just had a look and it doesn't appear to be.

**Reply by arvinxx** [reply]
Currently I don't want to support this. Due to client db have different id like server db. So if you import server db data to client db, it will make chaos. But after refactor the client db to pglite, it will be OK to import data from server db to client db, as pglite and server postgres is the same database.
